.storybook/main.js

```diff
@@ -37,6 +37,6 @@ module.exports = {
   },
   env: (config) => ({
     ...config,
-    REACT_APP_MAPBOX_API_KEY: process.env.REACT_APP_MAPBOX_API_KEY,
+    REACT_APP_MAPBOX_API_KEY: process.env.MAPBOX_API_KEY || process.env.REACT_APP_MAPBOX_API_KEY,
```

Comment:
Will there ever be a `REACT_APP_MAPBOX_API_KEY` env var since we renamed the secret?

Reply:
We renamed the secret in github ci, but keeping `REACT_APP_MAPBOX_API_KEY` here means we can reuse the env variable used in the create-react-app generated test-site otherwise our .env would be like this

```
MAPBOX_API_KEY=pk.1234
REACT_APP_MAPBOX_API_KEY=pk.1234
```

which isn't terrible, but seems like a better experience to set it once. fyi it is a create-react-app limitation to not be able to read env variables without their prefix